### PR TITLE
docs: Chinese translations updates (#1706)

### PR DESCRIPTION
**Pull request by tonels**
add Chinese translations updates (based issue #1706)

### CODE REVIEW

**Comment by tonels** [comment]
how can i resovle this ? @ohbus

File: localization/zh/filterer/README.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
+---
+layout: pattern
+title: Filterer
+folder: filterer
+permalink: /patterns/filterer/zh
+description: Design pattern that helps container-like objects to return filtered version of themselves.# short meta description that shows in Google search results
+language: zh
+categories:
+- Functional
+  tags:
```
Comment (iluwatar) [inline localization/zh/filterer/README.md:10]:
```suggestion
tags:
```

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Comment by iluwatar** [comment]
Ping @xiaod-dev, if you have time to review this?

**Comment by xiaod-dev** [comment]
> Ping @xiaod-dev, if you have time to review this?
9999999999999999999999999999ms.
wow. That's lot's of changes. It will take some times to review it.

**Review by xiaod-dev — CHANGES_REQUESTED** [review]
@tonels
please update the pic path in all files. Cause the directory structure has been changed. It should be like this:
`alt text` .
**Noted that if the pic path are correct, the pics should display normally.**

**Comment by iluwatar** [comment]
Closed due to inactivity

**Pull request closed without merging**